style.css

```diff
@@ -229,6 +322,40 @@ img {
   margin: 3.6rem;
 }
 
+.about-myself {
+  background-color: var(--color-secondary);
+  display: grid;
+}
+
+@media screen and (min-width: 76.8rem) {
+  .about-myself {
+    grid-template-columns: 1fr 1fr;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
- [ ] The about myself section is good, but on the desktop version, there are 2 blocks and the left one takes 40% and the right one the remaining 60% of the page's width. Kindly fix that width to match the Figma template.
 - [ ] Kindly add some margin on the left and right of this section to match the template and some padding on top with the semi-circle on the top right corner to match the template.
 - Your design:
 - 
<img width="1316" alt="image" src="https://user-images.githubusercontent.com/61919617/206236653-2ee19185-2c9e-4402-9ce3-70b96442b2b2.png">

 - Figma template:
 - 
<img width="953" alt="image" src="https://user-images.githubusercontent.com/61919617/206236730-7623544c-c5a5-4b6e-913f-4ad6ba116a2d.png">